sda-commons-client-jersey/src/main/java/org/sdase/commons/client/jersey/builder/AbstractBaseClientBuilder.java

```diff
@@ -24,7 +23,7 @@
  *
  * @param <T> the type of the subclass
  */
-abstract class AbstractBaseClientBuilder<T extends AbstractBaseClientBuilder> {
+abstract class AbstractBaseClientBuilder<T extends AbstractBaseClientBuilder<?>> {
```

Comment:
Isn't this a `T` as well?

```suggestion
abstract class AbstractBaseClientBuilder<T extends AbstractBaseClientBuilder<T>> {
```

Reply:
I was afraid of the recursion, but yes 😆 
Still works when I use `T`. Updated the PR.